app/assets/stylesheets/_shirts.scss

```diff
@@ -0,0 +1,21 @@
+div.shirt-banner {
+  margin-bottom: $base-spacing;
+  overflow-x: hidden;
+  white-space: nowrap;
+
+  a {
+    @include animation(marquee 20.0s linear infinite);
+    color: white;
+    display: inline-block;
+    font-style: italic;
+    font-weight: bold;
+    padding-left: 100%;
+    text-decoration: none;
+    text-transform: uppercase;
+  }
+}
+
+@include keyframes(marquee) {
+  0% { @include transform(translate(0, 0)); }
+  100% { @include transform(translate(-100%, 0)); }
+}
```

Comment:
On one hand, we're emulating a very old (relatively) school concept, similar to how the `<marquee>` is old.

On the other hand we're using all sorts of modern (relatively) tech to do it (http 1.1, rails, heroku) unlike `<marquee>`.

I can see it both ways, but it seems like you are for the `<marquee>` option, so I'll change it. :)